src/transformers/utils/sentencepiece_model_pb2.py

```diff
@@ -0,0 +1,1169 @@
+# flake8: noqa
```

Comment:
Need a copyright here, and what is this file's purpose (I don't understand the file name)?

Reply:
Is this file supposed to be used as it is?

Reply:
It's a generated file from sentence piece proto file, that's why no LICENSE.
its purpose it just to read `.spm` files (which are just proto files) so we can read the precompiled_normalizer binary string (which is not exposed by `spm` directly)